src/sdl2.nim

```diff
@@ -2167,7 +2167,7 @@ proc copyEx*(renderer: RendererPtr; texture: TexturePtr;
   ## `flip` `RendererFlip` value stating which flipping actions should be
   ## performed on the texture.
 
-proc clear*(renderer: RendererPtr): cint {.
+proc clear*(renderer: RendererPtr): SDL_Return {.
```

Comment:
https://wiki.libsdl.org/SDL2/SDL_RenderClear says the return type is C's `int`.

Reply:
Yes, the same for SDL_SetRenderDrawColor, SDL_SetRenderDrawBlendMode, SDL_SetWindowFullscreen, ... and the wrapper use SDL_Return for their result type.